lib/screens/home/staking/staking_delegation/staking_text_form_field.dart

```diff
@@ -28,6 +30,22 @@ class StakingTextFormField extends StatelessWidget {
           submit!();
         }
       },
+      onTap: () async {
```

Comment:
I mean, probably. I'll look into it.

Reply:
Just so that the conversation exists here too, I don't think we need to have the focus node stuff - just because these textfields are by themselves, and we don't need the node for macOS since we already have access to a hardware keyboard and there's no resizing or adjusting.